addons/l10n_ro_edi_stock/data/template_etransport.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template id="l10n_ro_template_etransport">
+        <eTransport
+            xmlns="mfp:anaf:dgti:eTransport:declaratie:v2"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="mfp:anaf:dgti:eTransport:declaratie:v2"
+            t-att-codDeclarant="interface.l10n_ro_edi_stock_company_id.vat.upper().replace('RO', '')"
```

Comment:
For this file, can you try to, as much as possible, do the complex operations in python? It's for making the XML as clean as possible.

`something.vat.upper().replace('RO','')` is appearing multiple times and could probably be refactored if we were to do it in python.